joomla/joomla-cms
# [3.x] Backport jQuery 3.5 security fixes

The recently released JQuery 3.5.x has fixed two XSS issues. As we can't upgrade to a newer jQuery version in 3.x for BC reasons, the security team suggest to manually patch our jQuery.

As this change might have unexpected b/c implications and the vulnerability already is public and well documented, I'm opening this ticket on the public tracker.

### Summary of Changes
Backported jQuery 3.5 security fixes.

### Testing Instructions
Apply patch, browse various parts of the site and make sure that no JS errors show up in the browser console.
